### PR TITLE
fix(style): fix gdpr history styles

### PR DESCRIPTION
**Pull request by ramniel**
image-20210421-150712

### CODE REVIEW

**Comment by andresmoschini** [comment]
Thanks @ramniel! could you add a couple of screenshot to the PR description?

File: src/components/Reports/GDPRPermissions/PermissionExpandableRow.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ const PermissionExpandableRow = ({
               <button
                 type="button"
                 onClick={() => setExpanded(!expanded)}
-                className={`dp-expand-results ${expanded && 'dp-open-results'}`}
+                className={`dp-expand-results${expanded ? ' dp-open-results' : ''}`}
```
Comment (andresmoschini) [inline src/components/Reports/GDPRPermissions/PermissionExpandableRow.js:66]: I understand what you did, but, even when a extra space will be rendered I prefer the following approach (if it works fine):

```suggestion
                className={`dp-expand-results ${expanded ? 'dp-open-results' : ''}`}
```

In my opinion it is cleaner. What do you think?

**Comment by andresmoschini** [comment]
Nice!

@GusBaamonde did you took a look?

**Comment by cbernat** [comment]
This is looking nice!
https://cdn.fromdoppler.com/doppler-webapp/build3046/#/subscribers/cbernat@makingsense.com/gdpr

image

I've noticed that each time a request is done we are printing it in the console, this is not so nice...

File: src/components/Reports/GDPRPermissions/PermissionExpandableRow.js (hunk 1 above)
Comment (ramniel) [reply src/components/Reports/GDPRPermissions/PermissionExpandableRow.js:66]: I agree with the proposed approach. It looks cleaner.

**Comment by andresmoschini** [comment]
🚀

**Comment by GusBaamonde** [comment]
👍

**Comment by cbernat** [comment]
👍 
@ramniel could you remove api response logging in another PR?

**Comment by andresmoschini** [comment]
What happened with this PR?

**Pull request merged**